build.gradle

```diff
@@ -20,6 +20,7 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-validation'
     implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
     implementation 'org.springframework.boot:spring-boot-starter-mail'
+    implementation 'org.springframework.boot:spring-boot-starter-webflux'
```

Comment:
Oh, why does it need webflux?

Reply:
That's because Spring RestTemplate is in maintenance mode and WebClient should be used instead.